src/include/miopen/batchnorm/solvers.hpp

```diff
@@ -133,6 +133,7 @@ struct BnFwdInference final : BatchnormSolver
 
     bool IsApplicable(const ExecutionContext& context,
                       const miopen::batchnorm::ProblemDescription& problem) const override;
+    bool IsDynamic() const override { return true; }

```

Comment:
@averinevg Why did we marked `BnFwdInference` as dynamic?

Reply:
I got it from [here](https://github.com/ROCm/MIOpen/blob/037eb6dc436b1cf405fe37f7a2304f4dcf5c29d0/src/include/miopen/solver.hpp#L76). Is there something wrong with this?